app/services/follow_recommendations_service.rb

```diff
@@ -83,14 +82,16 @@ def build_follow_graph(indirect_follows, direct_follow_ids)
     indirect_follow_map
   end
 
-  # Returns an array of default follows in the same JSON format as the public API using AccountSerializer
+  # Returns an array of default follows from V2 Suggestions API
   def generate_default_follows
```

Comment:
here we reuse the Mastodon `AccountSuggestions` and return those handles rather than the previous `(bart, mark, tsepiso)` from the past